src/github/mod.rs

```diff
@@ -712,6 +760,60 @@ impl std::fmt::Display for RepoDiff {
     }
 }
 
+
+
+enum OrgMembershipDiff {
```

Comment:
I wasn't sure if I could use `TeamDiff` as it was built on `GitHubTeam` which we want to avoid here. Created a new diff for this specific scenario